src/fparser/Fortran2003.py

```diff
@@ -115,10 +115,12 @@ def __new__(cls, string, parent_cls = None):
         elif cls not in parent_cls:
             parent_cls.append(cls)
         #print '__new__:',cls.__name__,`string`
-        match = cls.__dict__.get('match', None)
+        # Get the class' match method if it has one
+        match = cls.__dict__.get('match')
+        #print '__new__:',type(match), dir(cls)
```

Comment:
Is this commented out line really needed and useful? I suggest removing it if not.